Riot/Modules/Room/RoomViewController.m

```diff
@@ -4801,7 +4801,22 @@ - (void)roomInputToolbarViewDidOpenActionMenu:(RoomInputToolbarView*)toolbarView
 
 - (void)roomInputToolbarView:(RoomInputToolbarView *)toolbarView sendAttributedTextMessage:(NSAttributedString *)attributedTextMessage
 {
-    [self sendAttributedTextMessage:attributedTextMessage];
+    // "/me" command is supported with Pills in RoomDataSource. Other commands
+    // currently work with identifiers (e.g. ban, invite, op, etc).
+    NSString *message;
+    if (@available(iOS 15.0, *))
+    {
+        message = [PillsFormatter stringByReplacingPillsIn:attributedTextMessage mode:PillsReplacementTextModeIdentifier];
+    }
+    else
+    {
+        message = attributedTextMessage.string;
+    }
+
+    if ([message hasPrefix:kMXKSlashCmdEmote] || [self isIRCStyleCommand:message] == NO)
+    {
+        [self sendAttributedTextMessage:attributedTextMessage];
```

Comment:
I can't quite work out what the original bug was 🤔 , previously we would send all messages to `sendAttributedTextMessage` incl slash commands, so that behavior seems unchanged in this PR ...

Reply:
I see. In that case would you mind renaming the `isIRCStyleCommand` to something more explicit, for instance `sendAsIRCStyleCommandIfPossible`? It is not used in many places so the rename would be quick & safe. Alternatively we could have two separate methods (`isIRCStyleCommand` and `sendIRCStyleCommand`) but that would be too much unnecessary refactors I think

Reply:
(will change that a bit as we will also unnecessary build the string without pills in the "/me" case here)